sceptre/template_handlers/http.py

```diff
@@ -75,9 +77,28 @@ def _get_template(self, url):
         :rtype: str
         """
         self.logger.debug("Downloading file from: %s", url)
+        session = self._get_retry_session()
         try:
-            response = requests.get(url)
+            response = session.get(url, timeout=(2, 5))
```

Comment:
I think the retry settings should probably be configured as handler options. Without being an explicit option, I wonder what's the purpose of ALWAYS retrying?

Reply:
I've changed it so that users can pass the timeout and retry as handler options.